src/pyetm/models/saved_scenario.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+from datetime import datetime
+from typing import Any, Dict, Optional, TYPE_CHECKING
+from pydantic import Field, PrivateAttr
+from pyetm.models.base import Base
+from pyetm.clients import BaseClient
+from pyetm.services.scenario_runners.create_saved_scenario import (
+    CreateSavedScenarioRunner,
+)
+from pyetm.services.scenario_runners.update_saved_scenario import (
+    UpdateSavedScenarioRunner,
+)
+
+if TYPE_CHECKING:
+    from pyetm.models.scenario import Scenario
+
+
+class SavedScenarioError(Exception):
+    """Base saved scenario error"""
+
+
+class SavedScenario(Base):
+    """
+    Pydantic model for a MyETM SavedScenario.
+
+    A SavedScenario wraps an ETEngine session scenario and persists it in MyETM.
+    The response includes both SavedScenario metadata and the full nested Scenario.
+    """
+
+    id: int = Field(..., description="Unique saved scenario identifier in MyETM")
+    scenario_id: int = Field(..., description="Reference to ETEngine scenario")
+    title: str = Field(..., description="Title of the saved scenario")
+    description: Optional[str] = None
+    private: Optional[bool] = False
+    area_code: Optional[str] = None
+    end_year: Optional[int] = None
+    created_at: Optional[datetime] = None
+    updated_at: Optional[datetime] = None
+    scenario: Optional[Dict[str, Any]] = None
+
+    _scenario_model: Optional[Scenario] = PrivateAttr(None)
+
```

Comment:
Let's also add friendly methods that allows users to skip the `Scenario` object.

```python
SavedScenario.load(ss_id)
SavedScenario.new(scenario_id=12345)
```